### DISCUSSION TITLE
Is it possible to move window outside of main app window?

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [emilk/egui#959] [Q&A]
I'm running egui demo apps in macOS with carco.
I'm curious whether I can drag child windows outside of main app window like how normal native app works.
It seems all child components are stuck in main window like web app.
Can we overcome this?
Because some users can have multiple monitors.

I really want to dive to egui framework, I really like the performance and a lot of flexibility.
Sorry for bad english :)
thanks.

**Comment by emilk** [comment]
No, you cannot do that. Currently everything is stuck inside the same window.
